### ISSUE TITLE
pass search_layers as third arg?

### ISSUE THREAD

**Issue by missinglink** [pelias/wof-admin-lookup#58]
I'm not that familiar with this code, it looks like this might be an error?

the function accepts 3 args but the first call to `self.lookup` does not pass `search_layers` as the third arg

also a question, should we refactor this so that `callback` is the last arg as per the node style?

```
28    LocalPIPService.prototype.lookup = function lookup(centroid, callback, search_layers)
35    self.lookup(centroid, callback);
```

https://github.com/pelias/wof-admin-lookup/blob/master/src/localPipResolver.js#L35

**Comment by orangejulius** [comment]
Oh, that may indeed be a bug that would manifest itself as the configurable list of layers elsewhere in `wof-admin-lookup` having no effect. And yes, the interface should be changed to the standard format. There are many other improvements that we should probably make in these two repos.

**Comment by orangejulius** [comment]
@trescube you fixed this recently, right?

**Comment by trescube** [comment]
Yep, fixed in #89
